### PR TITLE
Major Danish language update

### PR DESCRIPTION
**Pull request by Regmos**
Hi Nicke.

Some days ago, i've send you a mail with an update for my Danish translation (version: 2.34.1).
I don't know if you have received it, so now I will try it this way.
I have corrected some typos, so please use this one.

I believe it's prettty good, but I'm not able to test it.
When i try to load it, I get an error ([Image](https://imgur.com/VjoqAM9)).

I have tried everything i my power, but nothing helps.
Validation in Visual Studio: No issues found
Validation online: Everything OK
Comparing to the base file in BeyondCompare: The syntax is identical.
Delete/comment out lines: Same result
And countless manual proofreadings, to no avail.

I'm using ScreenToGif portable,
and I have deleted everything, and downloaded a new, several times.

I really hope you can fix this.

Kind regards
Regmos

### CODE REVIEW

**Comment by NickeManarin** [comment]
Hello Regmos, I had a busy week.
The issue with the file sent by email was a few "-" characters in some lines.

I'm in the process of reorganizing the structure and moving this project to .NET 6, so I'm holding PR for a few days.

File: ScreenToGif/Resources/Localization/StringResources.da.xaml [hunk 1]
```diff
@@ -1021,26 +1227,33 @@
...
-    <s:String x:Key="S.Resize.CurrentProperties">Egenskaber</s:String>
+    <s:String x:Key="S.Resize.Difference">Forskel</s:String>
     <s:String x:Key="S.Resize.Dpi">Dpi</s:String>
-    <s:String x:Key="S.Resize.NewProperties">Nye egenskaber</s:String>
+    <s:String x:Key="S.Resize.Options">Indstillinger</s:String>
+    <s:String x:Key="S.Resize.Pixels">Pixel (px)</s:String>
+    <s:String x:Key="S.Resize.Percent">Procent (%)</s:String>
     <s:String x:Key="S.Resize.Dpi2">DPI:</s:String>
     <s:String x:Key="S.Resize.KeepAspect">Bevar størrelsesforhold.</s:String>
+    -<s:String x:Key="S.Resize.ScalingQuality">Kvalitet:</s:String>
```
Comment (NickeManarin) [inline ScreenToGif/Resources/Localization/StringResources.da.xaml:1252]: Here are the lines with issue. Just erase the "-" in lines 1252-1256 or I'll do it for you when merging, :)

Comment (Regmos) [reply ScreenToGif/Resources/Localization/StringResources.da.xaml]: Hi Nicke.
I've uploaded a new version to Github. It should be OK, but I still can't test it.
Now I get a whole different kind of error ([image](https://i.imgur.com/gb3uEUb.png)).
If you can use it, please do so, and I will take a look at it, when you publish the next STF update.
Kind regards 
Regmos

Comment (NickeManarin) [reply ScreenToGif/Resources/Localization/StringResources.da.xaml]: Thanks. I'll fix that one.

Comment (NickeManarin) [reply ScreenToGif/Resources/Localization/StringResources.da.xaml]: It was just an issue with the key `S.Recorder.Fps.Range`, there's other two keys with a similar name (`Fpm` and `Fph`) and the three were set as `Fps`.

Comment (Regmos) [reply ScreenToGif/Resources/Localization/StringResources.da.xaml]: Hi Nicki.
Thats it. Now everything should be OK
I've reviewed it, fixed a few things, and uploaded a new version.
Kind regards
Regmos

**Pull request merged**